### PR TITLE
Adding system tests for HappyBase Table.rows().

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

File: system_tests/bigtable_happybase.py [hunk 1]
```diff
@@ -232,6 +234,180 @@ def test_row_with_timestamp(self):
...
+        row2_data = {COL1: value3}
+
+        # Need to clean-up row1 and row2 after.
+        self.rows_to_delete.append(ROW_KEY1)
+        self.rows_to_delete.append(ROW_KEY2)
+        table.put(ROW_KEY1, row1_data)
+        table.put(ROW_KEY2, row2_data)
+
+        rows = table.rows([ROW_KEY1, ROW_KEY2])
+        rows.sort(key=_FIRST_ELT)
```
Comment (tseaver) [inline system_tests/bigtable_happybase.py]: `sorted` somehow feels more idiomatic to me:

```python
rows = sorted(table.rows([ROW_KEY1, ROW_KEY2]), key=FIRST_ELT)
```

File: system_tests/bigtable_happybase.py [hunk 2]
```diff
@@ -232,6 +234,180 @@ def test_row_with_timestamp(self):
...
+
+        # Need to clean-up row1 and row2 after.
+        self.rows_to_delete.append(ROW_KEY1)
+        self.rows_to_delete.append(ROW_KEY2)
+        table.put(ROW_KEY1, row1_data)
+        table.put(ROW_KEY2, row2_data)
+
+        # Filter a single column present in both rows.
+        rows_col1 = table.rows([ROW_KEY1, ROW_KEY2], columns=[COL1])
+        rows_col1.sort(key=_FIRST_ELT)
```
Comment (tseaver) [inline system_tests/bigtable_happybase.py]: Ditto `sorted`

File: system_tests/bigtable_happybase.py (hunk 1 above)
Comment (dhermes) [inline system_tests/bigtable_happybase.py]: There are 9 uses of `.sort()` right now:

```bash
$ git grep -n '\.sort' -- system_tests/bigtable_happybase.py
system_tests/bigtable_happybase.py:254:        rows.sort(key=_FIRST_ELT)
system_tests/bigtable_happybase.py:276:        rows.sort(key=_FIRST_ELT)
system_tests/bigtable_happybase.py:317:        rows_col1.sort(key=_FIRST_ELT)
system_tests/bigtable_happybase.py:328:        rows_col_fam1.sort(key=_FIRST_ELT)
system_tests/bigtable_happybase.py:340:        rows_col_fam_overlap1.sort(key=_FIRST_ELT)
system_tests/bigtable_happybase.py:348:        rows_col_fam_overlap2.sort(key=_FIRST_ELT)
system_tests/bigtable_happybase.py:370:        rows.sort(key=_FIRST_ELT)
system_tests/bigtable_happybase.py:387:        rows.sort(key=_FIRST_ELT)
system_tests/bigtable_happybase.py:395:        rows.sort(key=_FIRST_ELT)
```

Would you like me to change them all?

Also, why is one more idiomatic than the other? In terms of memory use, in-place sort is "better", though memory use is essentially irrelevant in our system tests.

**Comment by tseaver** [comment]
`sorted()` doesn't commit to having the source be an actual list.  If that consideration is moot, then `list.sort()` is slightly more efficient (for big enough lists to make the cost of allocation measurable).

**Comment by dhermes** [comment]
@tseaver PTAL

**Comment by dhermes** [comment]
@tseaver Any issues here?

**Comment by tseaver** [comment]
LGTM

**Pull request merged**